### DISCUSSION TITLE
In the callback function on_stream_readable,cannot call quic_stream_write twice

### DISCUSSION THREAD

**Discussion by liulujiao** [Tencent/tquic#357] [General]
I try to add a function quic_stream_write  in simple_server class.but in simple_client class, client_on_stream_readable function only recv the data sended by quic_stream_write first time,can not receive the second time data.so in one stream.can only send data one time?

**Comment by iyangsj** [comment]
Could you share your sample code？

**Reply by liulujiao** [reply]
Thank you ,I have solved the question, set the fin flag false,then quic_stream_write can send message twice.
